workshop/Lab1/README.md

```diff
@@ -0,0 +1,106 @@
+# Lab 1: Deploy an application with in-memory storage.
+
+Learn to deploy an application hosted within the IBM Kubernetes Service that stores
```

Comment:
Sounds a bit blunt... perhaps "In this section we'll..." or "In this lab we'll...."